qftasm.js

```diff
@@ -176,15 +176,15 @@ function step_code() {
         vals.push(val);
     }
     
+    PC[1] = PC[1] + 1;
     PC[0] = PC[1];
-    PC[1] = PC[0]+1;
-    if(RAMdisplay.indexOf(0) >= 0){ update_display(0); }

```

Comment:
To the contrary, "only updating the RAM display for address 0 when explicitly written to" was a bug that I was fixing by calling the same RAMwrite method that is normally used to update the display.  The `if(RAMdisplay.indexOf(0) >= 0){ update_display(0); }` code is duplicated inside of RAMwrite, which I call on the next line.